src/Gizmos/babylon.boundingBoxGizmo.ts

```diff
@@ -133,6 +163,7 @@ module BABYLON {
                         _dragBehavior.moveAttached = false;
                         box.addBehavior(_dragBehavior);
                         _dragBehavior.onDragObservable.add((event)=>{
+                            this.dragObservable.notifyObservers({});

```

Comment:
Is there a way to avoid the {} but also keep the option open to passing back additional data in the future?